.github/workflows/release.yml

```diff
@@ -22,13 +22,31 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2.3.4
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: olafurpg/setup-scala@v13
-      - run: sbt ci-release
+      - uses: olafurpg/setup-scala@v14
+
+      - name: Run sbt ci-release (produces war as well)
+        run: sbt ci-release
         env:
           PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
           PGP_SECRET: ${{ secrets.PGP_SECRET }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+
+      - name: Find WAR file
+        id: find_war
+        run: echo "WAR_PATH=$(find . -name '*.war' | head -n 1)" >> $GITHUB_ENV
+
+      - name: Create Draft Release
+        uses: softprops/action-gh-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          files: ${{ env.WAR_PATH }}
+          name: Draft Release for ${{ github.ref }}
+          draft: true
+          prerelease: false
+          fail_on_unmatched_files: true
+#          body_path: .github/release-body.md
```

Comment:
what to do with this one actually?

Reply:
@miroslavpojer / we will use this when he completes the release notes action.

Reply:
Yes, here is an example of usage.
![image](https://github.com/AbsaOSS/atum-service/assets/109138042/954955d9-f70a-4f17-94d4-395e2b7f8dc9)

There is used `body` instead of `body_path`.

Reply:
See GH action [repo](https://github.com/AbsaOSS/generate-release-notes). Use 0.1.1 Release.
See Readme for example of usage.

Reply:
Cool cool, thank you guys 👍

Reply:
Just FYI - released new version 0.2.0 of Release Notes generator GH action.